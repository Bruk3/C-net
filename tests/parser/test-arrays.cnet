int main() {
    int[3] a = new int[3];
    x[0] = 1;
    x[1] = 2;
    x[2] = 3;

    int[x[2] + 1] b = new int[4]{1,2,3,4};
    int[1] c = new int[5]{1};
}