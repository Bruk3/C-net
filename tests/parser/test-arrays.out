int main()
{
	int[] a = new int[3] = {};
	x[0] = 1;
	x[1] = 2;
	x[2] = 3;
	int[] b = new int[4] = {1, 2, 3, 4};
	int[] c = new int[5] = {1};
}


